docker/windows/install-dev.bat

```diff
@@ -0,0 +1,4 @@
+cd /home/iotedge/tool
```

Comment:
Can we create an install-dev file that works cross plat?  So we don't need two of them?

Reply:
Unfortunately not since nanoserver doesn't support .sh scripts